<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputWindow</class>
 <widget class="QDialog" name="OutputWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1187</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Вывод данных</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/rec/img/ico.ico</normaloff>:/rec/img/ico.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Выбор имени таблицы для просмотра даных</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="nameTable"/>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="selectCntStr"/>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="saveToCSV">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover { background-color: rgb(186, 189, 182) }
QPushButton:pressed { background-color: rgb(85, 87, 83) }</string>
     </property>
     <property name="text">
      <string>Сохранить в CSV формате</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="getDataBtn">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover { background-color: rgb(186, 189, 182) }
QPushButton:pressed { background-color: rgb(85, 87, 83) }</string>
     </property>
     <property name="text">
      <string>Получить данные в таблицу</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="delSelectedBtn">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover { background-color: rgb(186, 189, 182) }
QPushButton:pressed { background-color: rgb(85, 87, 83) }</string>
     </property>
     <property name="text">
      <string>Удалить выделенные элементы</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
